README.md

```diff
@@ -20,7 +20,7 @@ limitations under the License.
     <img alt="tool icon" src="https://raw.githubusercontent.com/neuralmagic/deepsparse/main/docs/source/icon-deepsparse.png" />
     &nbsp;&nbsp;DeepSparse Engine
   </h1>
-  <h3> Sparsity-aware neural network inference engine for GPU-class performance on CPUs </h3>
+  <h3> An inference runtime offering GPU-class performance on CPUs and APIs to integrate ML into your application. </h3>
```

Comment:
```suggestion
  <h3> An inference runtime offering GPU-class performance on CPUs and APIs to integrate ML into your application.</h3>
```

Reply:
taglines/subheadlines  don't normally get periods at the end as we haven't been doing that stylistically throughout the site.